components/item/MediaThumbnailGrid.vue

```diff
@@ -4,18 +4,23 @@
     class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
-    <b-img-lazy
+    <a
       v-for="(thumbnail, index) of thumbnails"
       :key="index"
-      :src="thumbnail.src"
       :class="thumbnailImgClass(thumbnail, index)"
       :data-about="thumbnail.about"
-      :data-qa="`media thumbnail #${index + 1}`"
-      thumbnail
-      alt=""
-      class="mb-2 mr-2 rounded-0"
-      @click.native="clickThumbnail(thumbnail.about)"
-    />
+      :data-qa="`link thumbnail #${index + 1}`"
+      href="#"
+      class="thumbnail-link mb-2 mr-2 rounded-"
+      @click="clickThumbnail(thumbnail.about)"
+    >
+      <b-img-lazy
+        :src="thumbnail.src"
+        :data-qa="`media thumbnail #${index + 1}`"
+        thumbnail
+        :alt="$t('mediaPreview.selectMedia', { src: thumbnail.src })"
```

Comment:
I am not sure if this is the best alt text. This could mean that a screen reader must read `Select media https://api.europeana.eu/api/v2/thumbnail-by-url.json?size=w200&type=IMAGE&uri=http%3A%2F%2Fwww.mimo-international.com%2Fmedia%2FGNM%2FIMAGE%2FMINe99_1309160825120_2.jpg`. 

I know that we left the text empty before, because we didn't find the best text yet. An alternative could be "Select 2nd (3th, 4th, etc) ... "

Reply:
@lbiedinger what are your thoughts on this, as I followed your advice by using the path?

Reply:
Yes, we wanted to have this in here for now as to at least pass the axe tests, but do a follow up.
One thing we were talking about is that it would be useful to know the 'format' of the media item, but doing so was pushing this work well out of scope for the original ticket here.

Reply:
Are you happy to merge this @lbiedinger ?